src/App.js

```diff
@@ -1,23 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
```

Comment:
chang it to :
`
import React from "react";
import { useEffect, useState } from "react";`